src/Symfony/Component/Console/Application.php

```diff
@@ -124,7 +123,14 @@ public function run(InputInterface $input = null, OutputInterface $output = null
             if (!$this->catchExceptions) {
                 throw $e;
             }
+        } catch (\Error $e) {
```

Comment:
Using `Throwable` would make the `catchErrors` flag catch both exceptions and errors, where there already is a dedicated flag for exceptions.
So, if we keep both flags (and I still think we need it currently, as I tried to explain in my previous comments, unless we find a better alternative :/ ), it should probably remains `\Error` here.